src/core/server/integration_tests/saved_objects/migrations/group5/dot_kibana_split.test.ts

```diff
@@ -173,135 +173,15 @@ describe('split .kibana index into multiple system indices', () => {
-            "fleet-proxy",
-            "fleet-setup-lock",
-            "fleet-uninstall-tokens",
-            "graph-workspace",
-            "guided-onboarding-guide-state",
-            "guided-onboarding-plugin-state",
-            "index-pattern",
-            "infra-custom-dashboards",
-            "infrastructure-monitoring-log-view",
-            "infrastructure-ui-source",
-            "ingest-agent-policies",
-            "ingest-download-sources",
-            "ingest-outputs",
-            "ingest-package-policies",
-            "ingest_manager_settings",
-            "inventory-view",
-            "kql-telemetry",
-            "legacy-url-alias",
-            "lens",
-            "lens-ui-telemetry",
-            "links",
-            "maintenance-window",
-            "map",
-            "metrics-data-source",
-            "metrics-explorer-view",
-            "ml-job",
-            "ml-module",
-            "ml-trained-model",
-            "monitoring-telemetry",
-            "observability-onboarding-state",
-            "osquery-manager-usage-metric",
-            "osquery-pack",
-            "osquery-pack-asset",
-            "osquery-saved-query",
-            "policy-settings-protection-updates-note",
-            "query",
-            "risk-engine-configuration",
-            "rules-settings",
-            "sample-data-telemetry",
-            "search-session",
-            "search-telemetry",
-            "security-rule",
-            "security-solution-signals-migration",
-            "siem-detection-engine-rule-actions",
-            "siem-ui-timeline",
-            "siem-ui-timeline-note",
-            "siem-ui-timeline-pinned-event",
-            "slo",
-            "slo-settings",
-            "space",
-            "spaces-usage-stats",
-            "synthetics-dynamic-settings",
-            "synthetics-monitor",
-            "synthetics-param",
-            "synthetics-privates-locations",
-            "tag",
-            "telemetry",
-            "threshold-explorer-view",
-            "ui-metric",
-            "upgrade-assistant-ml-upgrade-operation",
-            "upgrade-assistant-reindex-operation",
-            "uptime-dynamic-settings",
-            "uptime-synthetics-api-key",
-            "url",
-            "usage-counters",
-            "workplace_search_telemetry",
-          ],
-          ".kibana_so_search": Array [
-            "search",
-          ],
-          ".kibana_so_ui": Array [
-            "canvas-workpad",
-            "dashboard",
-            "visualization",
-          ],
-          ".kibana_task_manager": Array [
-            "task",
-          ],
-        }
-      `);
+      const typesMap = indicesInfo[`.kibana_${currentVersion}_001`].mappings?._meta?.indexTypesMap;
+
+      expect(Array.isArray(typesMap['.kibana'])).toEqual(true);
+      expect(typesMap['.kibana'].length > 50).toEqual(true);
+      expect(typesMap['.kibana'].includes('action')).toEqual(true);
```

Comment:
Checking that the map contains an exhaustive list of mappings is not necessary.
This test is broken everytime anyone adds a new SO type.
I've relaxed the constraint a bit.